prody/proteins/blastpdb.py

```diff
@@ -282,10 +297,12 @@ def showSequenceTree(hits):
             inp.write("\n")
     cmd = clustalw + " hits.fasta"
     os.system(cmd)
+    msa = parseMSA('hits.aln')
```

Comment:
I am not sure but please make sure that clustalw always creates 'hit.aln' as one of its output files.

Reply:
It does. We should probably also have a general clustalw function and then we'd have a prefix argument instead of 'hits'